Sources/CoreCommands/Options.swift

```diff
@@ -270,11 +270,28 @@ public struct BuildOptions: ParsableArguments {
                 visibility: .hidden))
     public var xcbuildFlags: [String] = []
 
+    @Option(name: .customLong("Xbuild-tools-swiftc", withSingleDash: true),
+            parsing: .unconditionalSingleValue,
+            help: ArgumentHelp(
+                "Pass flag to Swift compiler invocations for build-time executables (manifest and plugins)",
+                visibility: .hidden))
+    public var _buildToolsSwiftCFlags: [String] = []
+
     @Option(name: .customLong("Xmanifest", withSingleDash: true),
             parsing: .unconditionalSingleValue,
             help: ArgumentHelp("Pass flag to the manifest build invocation",
```

Comment:
Shouldn't the help message mention that this option is deprecated?

Reply:
When `--build-path` was deprecated (#4195), the help string was completely hidden. Since it was already hidden in this case, I thought that keeping the help string was not a big deal. As far as I can find, no option that has been deprecated had included so in the help message and the deprecation messages seemed to happen in `postprocessArgParserResult`.

Would it be better for you if I simply do a `help: .hidden` here?

Reply:
I still think that deprecated options should be marked as such in their help messages. If there's no established precedent we could start with this one. It's better to warn users before they consider adding this option to their CLI invocations, not only after the fact. But hiding them is probably the next best option.